lib/features/composer/presentation/composer_bindings.dart

```diff
@@ -89,43 +93,52 @@ class ComposerBindings extends BaseBindings {
       Get.find<DioClient>(tag: BindingTag.isolateTag),
       Get.find<Executor>(),
       Get.find<FileUtils>(),
-    ));
+    ), tag: composerId);
   }
 
   @override
   void bindingsDataSourceImpl() {
     Get.lazyPut(() => AttachmentUploadDataSourceImpl(
-      Get.find<FileUploader>(),
+      Get.find<FileUploader>(tag: composerId),
```

Comment:
Why we need a specific instance of FileUploader for each composer?

Reply:
And not found the put for this.

Reply:
> Why we need a specific instance of FileUploader for each composer?

Since it is only used in composer, we want to free it when not in use.

> And not found the put for this.

Please see `bindingUtils` method on the above